Temporary workaround for issue #30

Novack opened the pull request:
Small hack that will prevent notifications to be triggered more than once for the same threadId. Is a temporary workaround for Issue #30, that will offer more time to elaborate on a propper solution for the underlying issue.

Novack committed:
Merge remote-tracking branch 'skiff-org/main' into main

Novack committed:
Temp workaround for issue 30

Issue #30 temporarilily solved by this hack.

amilich commented on skiffWindowsApp/Skiff Desktop/MessageProcessor.cs:
```diff
@@ -48,6 +48,9 @@ internal void CoreWebView2_WebMessageReceived(object sender, CoreWebView2WebMess
                         var notificationsPayload = JsonSerializer.Deserialize<NotificationDataWrapper>(receivedMessage.Data.ToString());
                         foreach (var notification in notificationsPayload.NotificationData)
                         {
+                            if (_consumedThreads.Contains(notification.ThreadId))
```
I think this should be `messageID` based? you may want notifications for the same `threadID`

Novack replied:
IIRC all our desktop notification actions operate on threadIDs. Based on that, I not even implemented mailID as was not going to have any use, and introduce risk of mistakes. But if we do have it, I can do it.

Novack replied:
Oh, but I see your point on this one. I may want to look into the mailId to hide further notifications rather than the thread. Let me work on it a bit more.

Novack committed:
Fixed to work per message

Now the hack works on a per-message basis instead of per thread, so different mails of a thread will still trigger notifications.

Novack commented:
Oh right, sent the changes. Also added one missing fundamental line, adding the id to the list 🤦‍♂️

amilich reviewed (approved):
lgtm!

amilich commented:
Just merged it, great fix. I don't have my release computer with signing keys on it until a few days from now, so waiting on that :'(
